packages/icons-vue/tasks/createIconComponent.js

```diff
@@ -55,6 +55,7 @@ export default {
       };
     }
     if (data.class) {
+      svgData.class = svgData.class || {}; // may be no static class
```

Comment:
just wondering could the default value for `svgData.class` be an empty object as well? on L48:

```js
const svgData = {
  attrs,
  on: listeners,
  class: {},
};
```

Reply:
I don't think it will do any harm, I was just trying to make the minimal impact change.